### PR TITLE
Unpowered Doors Opened by Xenos Quickly

### PR DESCRIPTION
**Pull request by BobSlim**
## About The Pull Request

The delay on door opening only occurs if it is powered.

## Why It's Good For The Game

Humans open doors instantly with crowbars. That seems to be fine. 

Xenos currently open powered doors with a two second delay. That seems to be fine.

Thus; xenos should be able to instantly open doors if they are unpowered.

## Changelog
:cl:
tweak: Xenos immediately open unpowered doors.
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
I'd be in favor of making it a timed action for both humans and xenos.

File: code/game/objects/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -222,21 +222,23 @@
...
+				return FALSE
+			if(welded)
+				to_chat(M, "<span class='warning'>\The [src] is welded shut.</span>")
+				return FALSE
+	
+	if(density) //Make sure it's still closed
+		spawn(0)
+			open(1)
+			M.visible_message("<span class='danger'>\The [M] pries \the [src] open.</span>", \
+			"<span class='danger'>We pry \the [src] open.</span>", null, 5)
```
Comment (Rohesie) [inline code/game/objects/machinery/doors/airlock.dm:241]:
```suggestion
		open(1)
		M.visible_message("<span class='danger'>\The [M] pries \the [src] open.</span>", \
			"<span class='danger'>We pry \the [src] open.</span>", null, 5)
```
No need for the `spawn()` really.

File: code/game/objects/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -222,21 +222,22 @@
...
-		if(density) //Make sure it's still closed
-			spawn(0)
-				open(1)
-				M.visible_message("<span class='danger'>\The [M] pries \the [src] open.</span>", \
-				"<span class='danger'>We pry \the [src] open.</span>", null, 5)
+
+	if(hasPower())
+		M.visible_message("<span class='warning'>\The [M] digs into \the [src] and begins to pry it open.</span>", \
+		"<span class='warning'>We dig into \the [src] and begin to pry it open.</span>", null, 5)
+		if(do_after(M, 40, FALSE, src, BUSY_ICON_HOSTILE) && !M.lying_angle)
```
Comment (psykzz) [inline code/game/objects/machinery/doors/airlock.dm:229]:
```suggestion
		if(do_after(M, 4 SECONDS, FALSE, src, BUSY_ICON_HOSTILE) && !M.lying_angle)
```

**Comment by Pariah919** [comment]
Doors are a leading cause of death for benos. Good change.

**Pull request merged**